package.json

```diff
@@ -45,7 +41,6 @@
     "i18next-browser-languagedetector": "^7.0.1",
     "it-pipe": "^3.0.1",
     "it-pushable": "^3.2.1",
```

Comment:
I believe we also do not need `it-pipe`, `it-pushable` and `@multiformats/multiaddr` - double check against the other PRs